.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Been a while since i used the JetBrains IDEs, is it common to have these files checked into source?

Reply:
Not sure... they seem to be auto-generated and the IDE suggests adding them?

I can add the folder to `.gitignore` if you'd like.

Reply:
I just want to make sure they won't cause any conflicts or issues with others using IntelliJ or CLion or whatever. If they're helpful for other users then great, but if they could cause issues or we aren't sure then probably best to play it safe and add to the gitignore file